checkstyle/src/main.rs

```diff
@@ -0,0 +1,132 @@
+use clap::Parser;
+use regex::Regex;
+use std::fs;
+use std::path::{Path, PathBuf};
+use std::process;
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Args {
```

Comment:
Let's add a couple more arguments: `include` and `exclude`. You can see example usage here: https://github.com/vaticle/typedb-cloud/blob/development/test/behaviour/BUILD#L54

Reply:
This crate is maintained under the official rust language organisation on GitHub, so let's try using that: https://github.com/rust-lang/glob